### ISSUE TITLE
make plot of accuracy vs flops for various bayesian inference algorithms

### ISSUE THREAD

**Issue by murphyk** [probml/pyprobml#695]
The goal is to write JAX code that creates a figure similar to this for a 1d inference problem from [vol 2](https://probml.github.io/pml-book/book2.html) sec 7.5.

<img width="589" alt="Screen Shot 2022-03-14 at 3 08 21 PM" src="https://user-images.githubusercontent.com/4632336/158269206-10c7fc86-83dc-4743-bbd5-8e47c35166e9.png">

Rather than using the clutter example from Minka, feel free to change it to a more 'standard' model (anything that illustrates the speed-accuracy tradeoff).

**Comment by drabbit17** [comment]
I would like to work on this one if it's still free. Is any of the inference algorithms already been implemented through another ticket, or should they all be implemented from scratch for this one?

**Comment by murphyk** [comment]
This figure is just meant as an example. What would be better would be to take  the main algorithms already discussed in sex 7.4 (eg Laplace, HMC, ADVI, SMC) and make a timing comparison of the existing code when they are all applied to the same problem. The running example in 7.4 is just a 1d posterior (parameter of a beta-binomial model), since it is easy to visualize, and makes it easy to compare  to exact inference and a grid approximation. But that's too simple for a timing comparison. I would suggest Bayesian logistic regression instead (for varying N and D), where the error is wrt 'ground truth' posterior computed eg using Blackjax. (Actually pymc or nympyro may already have such a timing comparison...)
